src/electrum/server.rs

```diff
@@ -490,6 +501,27 @@ impl Connection {
         Ok(result)
     }
 
+    fn log_rpc_event(&self, entries: &Vec<(&str, Value)>) {
+        if let Some(_) = self.rpc_logging {
```

Comment:
Since this is now only ever called from the macro, maybe drop the `if` here?

Reply:
You're right. I removed that.